package.json

```diff
@@ -178,7 +178,8 @@
       "jsx-a11y/no-onchange": "off",
       "jsx-a11y/label-has-for": "off",
       "redux-saga/no-unhandled-errors": 0,
-      "jsx-a11y/anchor-is-valid": "off"
+      "jsx-a11y/anchor-is-valid": "off",
+      "jsx-a11y/no-autofocus": "off"
```

Comment:
👍 I agree with your commit comments on this one:

> a11y complains about autoFocus, so this commit turns off that rule as
well.
> There are cases where autoFocus decreases accessibility, but let's judge it buy ourselves, since this use of it does not cause any harm. 

A modal is definitely a case where it's important to set focus for accessibility. 

Related to this, the [wai-aria authoring spec for modals](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal) recommends placing focus back to the modal toggle when it closes. I noticed that before your changes, this was the case with the Create Blueprint button on the Blueprints page, but nowhere else. And after your changes, this is the case with the Create Image button on the Blueprints page, but nowhere else. Do you happen to know why that changed? And if it's easy to implement that behavior for all of them?

Also, I tried to do this exact same thing (i.e. `<input>` with `autofocus`) in the User Account modal, and could never get it to work. 😢

Reply:
You mean to focus on the button that actually opened the modal when closing the modal?
So if you want this behavior I just need to add this logic in the close function.

Regarding the autoFocus, in react it's autoFocus not autofocus. Maybe you did this wrong?

Reply:
Yes, by "toggle" I meant the button that opens the modal. If you can add that behavior, that would be great. 😄 👍

Reply:
I was not able to find a way to implement this. :/ If this is not a regression, can we merge without this feature?